auth/didvc/vc_issue_test.go

```diff
@@ -0,0 +1,71 @@
+package didvc_test
+
+import (
+	"net/http"
+	"runtime"
+	"testing"
+
+	"github.com/agiledragon/gomonkey/v2"
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
+
+	"github.com/machinefi/sprout/auth/didvc"
+	"github.com/machinefi/sprout/testutil"
+)
+
+func TestIssueCredential(t *testing.T) {
+	if runtime.GOOS == `darwin` {
```

Comment:
why we should skip macos

Reply:
gomonky has issue on macos because of permission, bus it's run good on Linux